ClearKeep/Scenes/Call/Views/VoiceCall.swift

```diff
@@ -0,0 +1,274 @@
+						.frame(width: Constant.paddingButtonNext, height: Constant.paddingButtonNext)
+					speakerIcon
+						.renderingMode(.template)
+						.foregroundColor(foregroundCrossIcon)
+				}
+			}
+			.frame(maxWidth: .infinity)
+		}
+		.frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .top)
+	}
+
+	var buttonCancel: some View {
+		VStack {
+			Button {
+				timeCalled = 0
+			} label: {
+				ZStack {
+					Circle()
+						.strokeBorder(backgroundEndCall, lineWidth: Constant.borderLineWidth)
+						.background(Circle().foregroundColor(backgroundEndCall))
+						.frame(width: Constant.paddingButtonNext, height: Constant.paddingButtonNext)
+					AppTheme.shared.imageSet.phoneOffIcon
+						.renderingMode(.template)
+						.foregroundColor(foregroundColorWhite)
+				}
+		}
+			Text("Call.Cancel".localized)
+				.font(AppTheme.shared.fontSet.font(style: .body2))
+				.foregroundColor(foregroundColorWhite)
+		}
+	}
+}
+
+// MARK: - Color func
+private extension VoiceCall {
+	var foregroundColorWhite: Color {
+		AppTheme.shared.colorSet.offWhite
+	}
+
+	var foregroundColorBlack: Color {
+		AppTheme.shared.colorSet.black
+	}
+
+	var foregroundColorGrey5: Color {
+		AppTheme.shared.colorSet.grey5
+	}
+
+	var foregroundColorGreyLight: Color {
+		AppTheme.shared.colorSet.greyLight
+	}
+
+	var foregroundCrossIcon: Color {
+		colorScheme == .light ? foregroundColorBlack : foregroundColorGreyLight
+	}
+
+	var foregroundBackButton: Color {
+		colorScheme == .light ? foregroundColorWhite : foregroundColorGreyLight
+	}
+
+	var backgroundGradientPrimary: LinearGradient {
+		LinearGradient(gradient: Gradient(colors: AppTheme.shared.colorSet.gradientPrimary), startPoint: .leading, endPoint: .trailing)
+	}
+
+	var backgroundEndCall: Color {
+		AppTheme.shared.colorSet.errorDefault
+	}
+
+	var backgroundButtonMute: Color {
+		isTappedMute == true ? foregroundColorWhite : foregroundColorWhite.opacity(0)
+	}
+
+	var backgroundButtonCamera: Color {
+		isTappedCamera == true ? foregroundColorWhite : foregroundColorWhite.opacity(0)
+	}
+
+	var backgroundButtonSpeaker: Color {
+		isTappedSpeaker == true ? foregroundColorWhite : foregroundColorWhite.opacity(0)
+	}
+
+	var muteIcon: Image {
+		isTappedMute ? AppTheme.shared.imageSet.microphoneOffIcon : AppTheme.shared.imageSet.microPhoneIcon
+	}
+
+	var cameraIcon: Image {
+		isTappedCamera ? AppTheme.shared.imageSet.videoOffIcon : AppTheme.shared.imageSet.videoIcon
+	}
+
+	var speakerIcon: Image {
+		isTappedSpeaker ? AppTheme.shared.imageSet.speakerOffIcon : AppTheme.shared.imageSet.speakerIcon
+	}
+}
+
+// MARK: - Private Func
+private extension VoiceCall {
+	func customBack() {
+		self.presentationMode.wrappedValue.dismiss()
+	}
+
+	func getTimer() -> String {
+		let minutes = Int(timeCalled) / 60 % 60
+		let seconds = Int(timeCalled) % 60
+		return String(format : "%02i:%02i", minutes, seconds)
+	}
+}
+
+struct VoiceCall_Previews: PreviewProvider {
+	static var previews: some View {
+        VoiceCall()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/9d22ca8cb862d8c613cfefddad57a36d8fff42d1/ClearKeep/Scenes/Call/Views/VoiceCall.swift#L272"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>VoiceCall.swift:272</code></td>
    </tr>
  </tbody>
</table>